### PR TITLE
feat: stop the card flashing in eventCard

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Without `onError` the eventCards will fallback to the image, while we need it to fallback if there is no image

### CODE REVIEW

File: client/src/components/EventCard.tsx [hunk 1]
```diff
@@ -98,6 +98,7 @@ export const EventCard: React.FC<EventCardProps> = ({ event }) => {
           src={event.image_url}
```
Comment (gikf) [inline client/src/components/EventCard.tsx:98]:
```suggestion
          src={
            event.image_url ||
            'https://cdn.freecodecamp.org/chapter/brown-curtain-small.jpg'
          }
```

**Review by gikf — COMMENTED** [review]
Could you post steps for reproducing the issue, or what is solved? After I've removed event _Event Image Url_, there's broken image displayed.
image

Maybe when `!event.image_url` then fallback image should be used as the image `src`. Suggestion for that below.

**Comment by Sboonny** [comment]
[rec-tab.webm](https://user-images.githubusercontent.com/88248797/217737092-712a4412-ea65-4e32-8c01-28cd06e27131.webm)

The issue is the fallback is shown while the image is lazy loading

File: client/src/components/EventCard.tsx (hunk 1 above)
Comment (Sboonny) [reply client/src/components/EventCard.tsx]: Works 👍, nice call

**Comment by Sboonny** [comment]
Did you remember the [image falling gracefully](https://discord.com/channels/692816967895220344/988495800852152450/1033992660559609896) discussion in discord, and made suggestion based on it? 🤣 

I have forgot about it completely

**Comment by gikf** [comment]
Thanks, this makes sense. The `fallbackSrc` and `fallbackStrategy` still might be useful though, for cases when image_url is provided, but cannot be loaded.

**Review by gikf — APPROVED** [review]

**Pull request merged**